### PR TITLE
Restaurant main skeleton

### PR DESCRIPTION
**Pull request by Mohamed-js**
In this project:

- Made a home page for the restaurant.
- Made a menu page for the restaurant.
- Made a contact page for the restaurant.

### CODE REVIEW

File: src/components.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+const text = (elem, text, className) => {
+  let t = document.createElement(`${elem}`);
+  t.textContent = `${text}`;
+  t.classList.add(`${className}`);
+  return t;
+};
+
+function tabs() {
+  const navbar = text("ul", "", "navbar");
+  for (var i = 0; i < arguments.length; i += 1) {
```
Comment (wathigo) [inline src/components.js]: **REQUIRED**
* Let's make use of es6 syntax. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/section/es6_requirement.png)).

File: src/components.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    let el = text("li", `${arguments[i]}`, "nav-item");
+    navbar.appendChild(el);
+  }
+  return navbar;
+}
+
+function menuItems(arr) {
+  const menu = text("section", "", "menu");
+  for (var i = 0; i < arr.length; i += 1) {
```
Comment (wathigo) [inline src/components.js:10]: **REQUIRED**
* Let's make use of es6 syntax. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/section/es6_requirement.png)).

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+page.appendChild(components.tabs('Home', 'Menu', 'Contact'))
+page.appendChild(components.createMain())
+let main = document.getElementById('main');
+
+const navItems = document.querySelectorAll('.nav-item')
+navItems.forEach(item => {
+    item.onclick = () => {
+        if(item.textContent === 'Home')     {var tab = home.home;}
+        if(item.textContent === 'Menu')     {var tab = menu.menu;}
+        if(item.textContent === 'Contact')  {var tab = contact.contact;}
```
Comment (wathigo) [inline src/index.js]: **REQUIRED**
* Let's make use of es6 syntax. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/section/es6_requirement.png)).

File: package.json [hunk 4]
```diff
@@ -9,6 +9,10 @@
   "author": "Mohamed Atef",
   "license": "ISC",
   "devDependencies": {
+    "babel-eslint": "^10.1.0",
+    "eslint": "^6.8.0",
+    "eslint-config-airbnb-base": "^14.1.0",
+    "eslint-plugin-import": "^2.20.2",
     "webpack": "^5.28.0",
     "webpack-cli": "^4.5.0"
   }
```
Comment (wathigo) [inline package.json:19]: **REQUIRED**
Necessary run common commands like build, development should be in package.json inside `scripts'. Here is a basic example ([screenshot from Pathwright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/scripts_requirement.png))
```js
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },
```

**Review by wathigo — CHANGES_REQUESTED** [review]
## CHANGES REQUIRES :o: 
Great job on this project. Let's make a couple of changes before we move on. See the comments :point_down: 

**After making the changes, submit another request for review.**

**Happy Coding :computer: 
Simon Wathigo**

**Review by arslanbisharat — APPROVED** [review]
# Approved :clap:

Hello Developer, :wave:

Thank You for submitting this Pull Request. I am thrilled to announce that your project has met all the minimum requirements and now it is approved. :clap:

I am wishing you very good luck with the next project. :smile:

Once again, Congratulations!:joy:

**Best,
Arslan Bisharat(PT-TSE)
Slack: @arslanbisharat**

**Pull request merged**